transformations/auth.py

```diff
@@ -14,42 +19,69 @@ def login():
         username = request.form['username']
         password = request.form['password']
         error = None
-        ldap_hostname = current_app.config['LDAP_HOSTNAME']
-        ldap_client = ldap.initialize(ldap_hostname)
-        ldap_client.set_option(ldap.OPT_REFERRALS, 0)
-        user_dn = "uid=%s,%s,%s" % (username, current_app.config['LDAP_USER_DN'], current_app.config['LDAP_BASE_DN'])
-        group_dn = "%s,%s" % (current_app.config['LDAP_GROUP_DN'], current_app.config['LDAP_BASE_DN'])
-        try:
-            ldap_client.protocol_version = ldap.VERSION3
-            ldap_client.simple_bind_s(user_dn, password)
-        except Exception as ex:
-            error = ex
+        user = None
+        if re.search(r'LDAP', current_app.config["AUTH_TYPE"], flags=re.IGNORECASE) or "LDAP_HOSTNAME" \
```

Comment:
speed wise it might we worth it to use `current_app.config["AUTH_TYPE"].lower() == 'ldap'`

Reply:
if I specify CLOWDER as AUTH_TYPE and specify LDAP_HOSTNAME it will use LDAP.

Reply:
Change to only use LDAP_HOSTNAME as a deciding factor if AUTH_TYPE isn't specified. Changed to not use RE search.